backend/execution_cache/src/admission_test.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/admission/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	fakeAdmissionReview = v1beta1.AdmissionReview{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "pods",
+			APIVersion: "v1",
+		},
+		Request: &v1beta1.AdmissionRequest{
+			UID: "123",
+		},
+		Response: &v1beta1.AdmissionResponse{
+			UID: "123",
+		},
+	}
+)
+
+func fakeAdmitFunc(req *v1beta1.AdmissionRequest) ([]patchOperation, error) {
+	operation := patchOperation{
+		Op:    "add",
```

Comment:
I think you should use `Add` (or `OperationTypeAdd`, if you like the renaming) directly here